src/components/network/config-tables.js

```diff
@@ -31,6 +34,8 @@ export const TABLES_DEFINITIONS = {
     VOLTAGE_LEVELS: {
         index: 1,
         name: 'VoltageLevels',
+        resource: equipements.voltageLevels,
```

Comment:
is this undefined (and it works because the other is equipements.vol which is also undefined) ?

Reply:
yeah there is something missing... like an entry in equipements, should be the key in the previous comment